services/galley/test/integration/Main.hs

```diff
@@ -72,31 +68,30 @@ main = withOpenSSL $ runTests go
               (pathsConsistencyCheck . treeToPaths . compile $ Galley.API.sitemap),
           API.tests setup
         ]
+    getOpts :: FilePath -> FilePath -> IO TestSetup
     getOpts gFile iFile = do
       m <- newManager tlsManagerSettings {managerResponseTimeout = responseTimeoutMicro 300000000}
-      let local p = Endpoint {_epHost = "127.0.0.1", _epPort = p}
-      gConf <- handleParseError =<< decodeFileEither gFile
-      iConf <- handleParseError =<< decodeFileEither iFile
-      -- FUTUREWORK: we don't support process env setup any more, so both gconf and iConf
```

Comment:
I took the oppurtunity to solve this FUTUREWORK section.  I'm not sure  if there is a ticket open for "unifying the integration test suites into a single library". I don't fully understand what that would mean :)